### PR TITLE
feat(core): set display currency from country on account create

### PR DESCRIPTION
**Pull request by vindard**
## Description

Builds on https://github.com/GaloyMoney/price/pull/67, takes over #4271 to address #4252

**Note:** I deliberately didn't propagate this change all the way to the GraphQL currencies query. Can do this if we think we really need it

### CODE REVIEW

File: core/api/src/app/prices/display-currency-from-country-code.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { PriceService } from "@/services/price"
+
+export const displayCurrencyFromCountryCode = async (
```
Comment (vindard) [inline core/api/src/app/prices/display-currency-from-country-code.ts:3]: I implemented this at app layer, but is it something that could/should live in price service instead?

**Review by dolcalmi — CHANGES_REQUESTED** [review]

File: core/api/src/app/prices/display-currency-from-country-code.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  countryCode: string,
+): Promise<DisplayCurrency | false | ApplicationError> => {
```
[the remark this answers is not in the document]
Comment (vindard) [reply core/api/src/app/prices/display-currency-from-country-code.ts]: Refactored here https://github.com/GaloyMoney/galoy/pull/4309/commits/fccc7cf45bc8a4df664f38bd6ce81ecf49bfd8df

File: core/api/src/app/accounts/create-account.ts [hunk 3]
```diff
@@ -51,6 +55,12 @@ const initializeCreatedAccount = async ({
   account.statusHistory = [{ status: config.initialStatus, comment: "Initial Status" }]
   account.level = config.initialLevel
 
+  if (countryCode) {
+    const displayCurrency = await displayCurrencyFromCountryCode(countryCode)
```
[the remark this answers is not in the document]
Comment (vindard) [reply core/api/src/app/accounts/create-account.ts]: Refactored here https://github.com/GaloyMoney/galoy/pull/4309/commits/fccc7cf45bc8a4df664f38bd6ce81ecf49bfd8df

File: core/api/src/app/prices/display-currency-from-country-code.ts (hunk 1 above)
Comment (vindard) [reply core/api/src/app/prices/display-currency-from-country-code.ts]: Moved to domain https://github.com/GaloyMoney/galoy/pull/4309/commits/fccc7cf45bc8a4df664f38bd6ce81ecf49bfd8df

**Pull request merged**